.github/workflows/generate-linter.yml

```diff
@@ -26,12 +26,9 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
-      - name: Setup Go
-        uses: actions/setup-go@v4
-        with:
-          go-version: ${{ matrix.goVersion }}
       - name: build templates
-        run: go run main.go create -n ${{ matrix.framework }} -f ${{ matrix.framework}}
+        run: |
+          script -q /dev/null -c "go run main.go create -n ${{ matrix.framework }} -f ${{ matrix.framework}}" /dev/null
```

Comment:
Hm - this does look a bit hacky?
@mimatache i will leave you to look at this and give some suggestions

Reply:
@Melkeydev This is a good workaround for entering interactive mode with the application, though it is a hack around a GitHub limitation. 
Another option would be to simply not enter interactive mode for non interactive calls to the program. I have opened #96 for this. 
You can choose which you would prefer.